content/blog/using-jina-embeddings-haystack/index.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: blog-post
+title: 'Using Jina Embeddings v2 with Haystack 2.0 pipelines to summarize legal documents'
+description: Learn how to use the Jina v2 Embedding models in a RAG pipeline with our new Haystack integration.
+featured_image: thumbnail.png
+images: ["blog/using-jina-embeddings-haystack/thumbnail.png"]
+alt_image: The logos for Haystack and Jina AI sit against a blue background with cartoonish art of a man and a woman holding a magnifying glass up to a large clipboard, next to a gavel.
+date: 2024-01-08
+last_updated: 2024-01-08
+toc: True
+authors:
+  - Tilde Thurium
+tags: ["Community", "Embeddings", "Open Source", "Integration"]
+cookbook: jina-embeddings-v2-legal-analysis-rag.ipynb
+---
+
+[Jina.ai](https://jina.ai/) recently upgraded and expanded the capabilities of their previous embedding model in a v2 release. 
```

Comment:
Can you please link "`v2 release`" to https://jina.ai/news/jina-ai-launches-worlds-first-open-source-8k-text-embedding-rivaling-openai/ ?